modules/backend/layouts/_head.htm

```diff
@@ -50,6 +55,10 @@
     <link href="<?= $script . '?v=' . $coreBuild; ?>" rel="preload" as="script" importance="high">
 <?php endforeach; ?>
 
+<?php foreach ($fonts as $font) : ?>
+    <link href="<?= $font; ?>" rel="preload" as="font" type="font/woff2" crossorigin importance="high">
```

Comment:
Why is the font type specified as woff2 here hardcode when the font itself is woff? Also, why is it hardcoded instead of depending on the file extension?

Reply:
Done.